src/components/Asset/Edit/FormActions.tsx

```diff
@@ -23,7 +24,12 @@ export default function FormActions({
       >
         Submit
       </Button>
-      <Button style="text" onClick={() => setShowEdit(false)}>
+      <Button
+        style="text"
+        onClick={() => {
+          router.push(`/asset/${did}`)
```

Comment:
do we really need to use the router here directly, or can we use Next.js `Link` component? Trying to keep it consistent how we do internal linking, and `Link` has a bunch of more advantages

Reply:
used it first but that needed some styling adjustments on the cancel buttons that i wanted to avoid 😄 , going to replace the router.push with link and add the proper styling